app/controllers/social_previews_controller.rb

```diff
@@ -26,7 +28,7 @@ def listing
 
   def organization
     @user = Organization.find(params[:id])
-
+    @tag_badges = [] # Orgs don't have badges, but they could!
```

Comment:
`Badge.none` would be more idiomatic, since the rest of the calls return an `ActiveRecord::Relation` instead of an array